packages/block-editor/src/store/actions.js

```diff
@@ -1603,3 +1603,15 @@ export function setHasControlledInnerBlocks(
 		clientId,
 	};
 }
+
+/**
+ * Action that sets whether a block has controlled inner blocks.
+ *
+ * @param {Record<string,boolean>} updates The block's clientId.
```

Comment:
There's a mismatch between the English copy, which assumes a single block — thus suggesting an argument list like `( clientId, isVisible )` — and the type of `updates` — which suggests batching.

From the rest of the PR, I would guess that the batching is preferred, and so the fix would be to update the docstring:

_Action that sets whether given blocks are visible on the canvas._

and the param description:

_For each block's clientId, its new visibility setting._